lib/SILOptimizer/Mandatory/TFLowerGraph.cpp

```diff
@@ -1600,27 +1617,29 @@ GLStatus TFGraphLowering::visitTFDataset(BuiltinInst *inst) {
   return GLStatus::Success;
 }
 
-// TODO: It is wasteful to serialize functions into a proto and deserialize it
-// here. Revisit the long-term design.
-bool TFGraphLowering::copyGraphFunctions(const std::vector<char> &graphDefProto,
-                                         SILLocation loc) {
-  // Create a tmp graph to host all graph funcs in `graphDefProto`.
-  TF_Graph *tmpGraph = TF_NewGraph();
-  SWIFT_DEFER { TF_DeleteGraph(tmpGraph); };
-
-  TF_Buffer graphDefBuf{graphDefProto.data(), graphDefProto.size(), nullptr};
-  auto *graphDefOptions = TF_NewImportGraphDefOptions();
-  TF_GraphImportGraphDef(tmpGraph, &graphDefBuf, graphDefOptions, status);
-  TF_DeleteImportGraphDefOptions(graphDefOptions);
-  if (checkStatus(loc))
-    return true;
+static bool checkStatus(SILFunction &fn, SILLocation loc, TF_Status *status,
```

Comment:
Should the other checkStatus call this one?  Please add doc comment etc.

Reply:
Added a comment.

The other checkStatus() calls a member method internalError(), and it's only 3 lines of code, so I did not touch that.